### DISCUSSION TITLE
Just a little help :))

### DISCUSSION THREAD

**Discussion by numairawan** [jimmywarting/StreamSaver.js#284] [Q&A]
Thanks for help.

**Comment by jimmywarting** [comment]
if you are downloading a zip from a server then i really recommend you to use the server if at all possible first and formost

can't stretch how important this is... it have better support in all browsers, browsers can pause, resume and also retry failed attempts and continue from where it left of if you also provide a `Accept-Ranges: bytes` header along with a `content-length` 

streamsaver and other alike is mostly just for client side generated content.
